validation/management/commands/writetranscriptions.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+# Copyright 2021 Jolene Poulin <jcpoulin@ualberta.ca>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+import sqlite3
+import os
+import shutil
+from contextlib import closing
+
+from django.core.management.base import BaseCommand, CommandError  # type: ignore
+
+
+class Command(BaseCommand):
+    """
+    Takes all extracted audio files in .wav format
+    and writes transcription files for them that can
+    then be used by Persephone and Simple4All.
+    Files are created in the ./audio directory and stored by speaker
+    """
+
+    help = "creates transcription files for Persephone and Simple4All"
+
+    def handle(self, *args, **options):
+        # Change this if the raw audio files exist elsewhere!
+        _path = "./audio"
+
+        # Generate this folder by running:
+        # python3 manage.py importrecordings --wav --skip-db
+        audio_dir = Path(_path)
+        for audio_file in os.listdir(audio_dir):
```

Comment:
```suggestion
        for audio_file in audio_dir.iterdir():
```

It also amuses me that [the example in the docs](https://docs.python.org/3/library/pathlib.html#pathlib.Path.iterdir) just says `: child`, like it's disappointed with the filesystem.

Reply:
![image](https://user-images.githubusercontent.com/28357720/107557638-1e72b600-6b97-11eb-96cc-5c070f691454.png)